core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -1233,6 +1233,18 @@ public static boolean isValidPropertyKey(String key) {
     return validProperties.contains(key) || isKeyValidlyPrefixed(key);
   }
 
+  /**
+   * Checks if the given property prefix is valid. A valid property prefix is equal to some prefix
+   * defined in this class.
+   *
+   * @param prefix
+   *          property prefix
+   * @return true if prefix is valid (recognized)
+   */
+  public static boolean isValidPropertyPrefix(String prefix) {
+    return validPrefixes.contains(prefix);
+  }
+
```

Comment:
This method isn't necessary, since this loop can be avoided.
```suggestion
```

Reply:
Removed